src/main/java/com/metamx/metrics/GarbageCollectors.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2016 Metamarkets Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.metamx.metrics;
+
+import com.metamx.common.logger.Logger;
+
+import java.lang.management.GarbageCollectorMXBean;
+import java.lang.management.ManagementFactory;
+import java.util.ArrayList;
+import java.util.List;
+
+final class GarbageCollectors
```

Comment:
Maybe it's better not to use `ManagementFactory.getPlatformMXBeans`. Since we already have `JStatData` we can get GarbageCollector names from it. It's `sun.gc.collector.0.name` and `sun.gc.collector.1.name` counters.